### PR TITLE
GEM - updated GEB dataformat

### PR DESCRIPTION
**Pull request by jshlee**
corresponds to changes made in cms-gem-daq-project/gem-light-dqm
changes ZeroSup -> ZeroSupWordsCnt
added Optohybrid BC and EC

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: EventFilter/GEMRawToDigi/interface/GEBdata.h [hunk 1]
```diff
@@ -65,20 +65,24 @@ namespace gem {
       m_Vwt = 0x0fff & (word >> 36);  /*!<VFAT word count*/
       m_InFu = 0x0f & (word >> 35);   /*!<InFIFO underflow*/
       m_Stuckd = 0x01 & (word >> 34); /*!<Stuck data*/
+      m_OHBC = 0x0fff & (word >> 20); /*!OH BC*/
```
Comment (perrotta) [inline EventFilter/GEMRawToDigi/interface/GEBdata.h]: Please update the comment at L60 with the two new contents

Comment (jshlee) [reply EventFilter/GEMRawToDigi/interface/GEBdata.h]: done

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
(No reason to wait for the tests to finish, since the last update was only in a comment line)
- Packed header and traiiler words updated in accordance with the description
- Jenkins tests pass

**Comment by fabiocos** [comment]
+1

**Pull request merged**